bin/main.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+'''Copyright (C) 2017 Collabora
+This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this
+file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+Authors: Shane Fagan - shane.fagan@collabora.com
+'''
+
+import datetime
+import sys
+import os
+from time import sleep
+from threading import Thread
+sys.path.append(os.path.dirname(__file__))
+RULES_PATH = os.path.abspath(os.path.join('..', 'rules', 'simple'))
+
+
+class Main():
+    '''
+        Class to handle states
+    '''
+    def __init__(self):
+        pass
+
+
+def __log_message(message):
+    '''
+        Internal logging for debug
+    '''
+    with open("messages.log", "a", encoding="utf-8") as f:
```

Comment:
We do not need a messages log file for now, that's for the future, let's keep this simple (and our work environments clean ;)).